CHANGES.md

```diff
@@ -13,6 +13,16 @@ development source code and as such may not be routinely kept up to date.
 
 # __NEXT__
 
+This release adds support for Python version 3.12.
+([#TBD](https://github.com/nextstrain/cli/pull/TBD))
+
+Note that this Python version support only matters if you're installing
+Nextstrain CLI from PyPI or Bioconda
+([c.f.](https://docs.nextstrain.org/projects/cli/en/8.3.0/installation/)).  It
```

Comment:
This link shouldn't be version-specific here. It should at least be version-agnostic

```suggestion
([c.f.](https://docs.nextstrain.org/projects/cli/page/installation/)).  It
```

or, better yet, reflect the version it's released as



```suggestion
([c.f.](https://docs.nextstrain.org/projects/cli/en/__NEXT__/installation/)).  It
```

(See c05b0447587fdd45375badee924e162ebdb65fd2 for how that works.)